resources/views/entities/show.blade.php

```diff
@@ -1,58 +1,62 @@
 <x-app-wide-layout>
     <x-slot name="title">{{ $entity->name }}</x-slot>
     <x-slot name="header">
-        <h1>
+        <h1 id="entity">
             {{ $entity->name }}
         </h1>
     </x-slot>
 
     <p>{{ $entity->locality }}, {{ get_region_name($entity->region, ["CA"], locale()) }}</p>
 
-    <div class="tabs flow" x-data="tabs(window.location.hash ? window.location.hash.substring(1) : 'about')" x-on:resize.window="enabled = window.innerWidth > 1023">
-        <h2 x-show="!enabled">{{ __('Contents') }}</h2>
-        <ul x-bind="tabList">
-            <li x-bind="tabWrapper"><a href="#about" x-bind="tab">{{ __('entity.section_about') }}</a></li>
-            <li x-bind="tabWrapper"><a href="#accessibility-and-inclusion" x-bind="tab">{{ __('entity.section_accessibility_and_inclusion') }}</a></li>
-            <li x-bind="tabWrapper"><a href="#projects" x-bind="tab">{{ __('entity.section_projects') }}</a></li>
-        </ul>
-
-        <div class="flow" id="about" x-bind="tabpanel">
-            <h2>{{ __('entity.section_about') }}</h2>
+    <div class="has-nav-secondary">
+        <nav class="secondary" aria-labelledby="entity">
+            <ul role="list">
+                <x-nav-link :href="localized_route('entities.show', $entity)" :active="request()->routeIs(locale() . '.entities.show')">{{ __('About') }}</x-nav-link>
+                <x-nav-link :href="localized_route('entities.show-accessibility-and-inclusion', $entity)" :active="request()->routeIs(locale() . '.entities.show-accessibility-and-inclusion')">{{ __('Accessibility and inclusion') }}</x-nav-link>
+                <x-nav-link :href="localized_route('entities.show-projects', $entity)" :active="request()->routeIs(locale() . '.entities.show-projects')">{{ __('Projects') }}</x-nav-link>
```

Comment:
Where the route is defined it looks like it's just named `entities.show-projects`.  Is there somewhere that's updating the names with the locale and/or is it possible to set the `:active` state with just the name and not including the locale in the `routeIs` call?

Reply:
The actual route name with locale is retrieved using `localized_route`— basically it prepends a locale code. So the effective route name is `en.entities.show`, not `entities.show`. The locale does need to be included in the `routeIs` call as a result.